.gitignore

```diff
@@ -12,4 +12,4 @@ config/*
 composer.lock
 .idea
 .DS_Store
-
+phpstan.neon
```

Comment:
This ignores all phpstan.neon, we might want to have one of those in some subfolder, better ignore
/phpstan.neon